### PR TITLE
Hawaii Tax bill H.B. 2404 H.D. 1, S.D.1, C.D.1 into law, becoming Act 46 SLH 2024

### LINKED ISSUE #4681: Hawaii tTx bill H.B. 2404 H.D. 1, S.D.1, C.D.1 into law, becoming Act 46 SLH 2024
[Bill text](https://data.capitol.hawaii.gov/sessions/session2024/bills/HB2404_CD1_.HTM)

[State Analysis](https://tax.hawaii.gov/blog/blog16-estimated-impacts-of-2024-tax-cut-bill/)

### PR DESCRIPTION
**Pull request by PavelMakarchuk**
Fixes #4681

### CODE REVIEW

File: policyengine_us/parameters/gov/states/hi/tax/income/rates/surviving_spouse.yaml [hunk 1]
```diff
@@ -1,60 +1,95 @@
-description: Hawaii taxes widow filers at these rates by taxable income.
+description: Hawaii taxes widow(er) filers at these rates by taxable income.
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/hi/tax/income/rates/surviving_spouse.yaml:1]:
```suggestion
description: Hawaii taxes surviving spouse filers at these rates by taxable income.
```

File: policyengine_us/parameters/gov/states/hi/tax/income/rates/surviving_spouse.yaml [hunk 2]
```diff
@@ -1,60 +1,95 @@
...
-      2017-01-01: 0.11
+      2018-01-01: 0.11
 
 metadata:
   threshold_unit: currency_USD
   rate_unit: /1
   threshold_period: year
   type: marginal_rate
-  label: Hawaii widow(ers) income tax rate
+  label: Hawaii widow(er) income tax rate
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/hi/tax/income/rates/surviving_spouse.yaml:90]:
```suggestion
  label: Hawaii surviving spouse income tax rate
```

**Review by MaxGhenis — CHANGES_REQUESTED** [review]
any legal code for the existing rules? if not could you add 2023 forms as reference?

**Comment by PavelMakarchuk** [comment]
> any legal code for the existing rules? if not could you add 2023 forms as reference?

I added the 2023 form for the SD but for the tax rate, the link to the tax table is most recent document

**Comment by MaxGhenis** [comment]
could you confirm standard deduction and rates aren't in the legal code? if so please leave a comment

**Comment by PavelMakarchuk** [comment]
> could you confirm standard deduction and rates aren't in the legal code? if so please leave a comment

Found the legal code

File: policyengine_us/parameters/gov/states/hi/tax/income/deductions/standard/amount.yaml [hunk 3]
```diff
@@ -1,21 +1,52 @@
...
   - filing_status
   reference:
   - title: 2022 Hawaii Tax Form Instruction Booklet line 23
     href: https://files.hawaii.gov/tax/forms/2022/n11ins.pdf#page=20
+  - title: 2023 Hawaii Tax Form Instruction Booklet line 23
+    href: https://files.hawaii.gov/tax/forms/2023/n11ins.pdf#page=20
+  - title: Tax Bill H.B. 2404 H.D. 1, S.D.1, C.D.1 Section 235—2.4 (a)(2)
+    href: https://data.capitol.hawaii.gov/sessions/session2024/bills/HB2404_CD1_.HTM
+  - title: Hawaii Revised Statutes §235-2.4 (a)(2)
+    href: https://data.capitol.hawaii.gov/sessions/session2024/bills/HB2404_CD1_.HTM
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/hi/tax/income/deductions/standard/amount.yaml:16]: wrong link?

**Review by MaxGhenis — CHANGES_REQUESTED** [review]
it looks like the legal code doesn't yet reflect this bill's passage. please leave a comment in each reference noting this.

File: policyengine_us/parameters/gov/states/hi/tax/income/deductions/standard/amount.yaml (hunk 3 above)
Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/hi/tax/income/deductions/standard/amount.yaml]: yep, changed

**Review by MaxGhenis — APPROVED** [review]

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**